statsmodels/stats/tests/test_power.py

```diff
@@ -727,28 +727,6 @@ def test_power_solver_warn():
     nip.start_bqexp['nobs1'] = {'upp': 50, 'low': -20}
     val = nip.solve_power(0.1, nobs1=None, alpha=0.01, power=pow_, ratio=1,
                           alternative='larger')
-    import scipy
-    if scipy.__version__ < '0.10':
```

Comment:
AFAIR, this shouldn't be deleted, the problem was with 0.12 which failed compared to older versions.
This needs checking if it works with recent scipy.
i.e. need to check changes in scipy optimize.

IIRC